src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java

```diff
@@ -791,7 +796,21 @@ public static HearingDayDetailsEntity hearingDayDetailsEntity() {
         entity.setVenueId("venueId");
 
         HearingAttendeeDetailsEntity attendee = new HearingAttendeeDetailsEntity();
-        entity.setHearingAttendeeDetails(List.of(attendee));
+        attendee.setId(2000000000L);
```

Comment:
Use a constant, or final local var ?

Reply:
Updated in f258ba42a8e3a5c96b9786cca080d3344e0f71c8